mslib/msui/msui_mainwindow.py

```diff
@@ -457,6 +447,106 @@ def __init__(self, local_operations_data=None, tutorial_mode=False, *args):
         self.config_editor = None
         self.local_active = True
         self.new_flight_track_counter = 0
+        edit = editor.ConfigurationEditorWindow(self)
+        self.config_for_gui = edit.last_saved
+        self.config_for_gui["automated_plotting_flights"].clear()
+        self.config_for_gui["automated_plotting_hsecs"].clear()
+        self.config_for_gui["automated_plotting_vsecs"].clear()
+        self.config_for_gui["automated_plotting_lsecs"].clear()
+        map_sections = {
```

Comment:
ToDo: this needs to be done differently. The map_sections is user specific. This is usually changed for the region of the aircraft mission.

@Preetam-Das26 Why is it needed when we can use the default from https://github.com/Open-MSS/MSS/blob/GSOC2024-rebased-autoplot/mslib/utils/config.py#L76

Reply:
https://github.com/Open-MSS/MSS/pull/2550